src/Keboola/StorageApi/Client.php

```diff
@@ -361,7 +361,11 @@ public function linkBucket($name, $stage, $sourceProjectId, $sourceBucketId, $di
             $options['displayName'] = $displayName;
         }
 
-        $result = $this->apiPost("buckets", $options);
+        if ($async) {
+            $options['async'] = $async;
+        }
+
+        $result = $this->apiPost("buckets", $options, $async);
```

Comment:
Mělo by sa posílat url takto sa to pošle v post params.

Reply:
Diky 🙏 upravil som to